google-cloud-bigtable/lib/google/cloud/bigtable/chunk_reader.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
```

Comment:
This file needs the copyright header. Be sure to leave two empty spaces afterwards so YARD doesn't use the copyright text as documentation for the `Google` module.

Reply:
Will add copywriter to all files.